tensornetwork/backends/jax/jax_backend.py

```diff
@@ -55,10 +55,17 @@ def __init__(self, dtype: Optional[np.dtype] = None) -> None:
     jsp = libjax.scipy
     self.name = "jax"
     self._dtype = np.dtype(dtype) if dtype is not None else None
+    self.jax_precision = precision if precision is not None else libjax.lax.Precision.DEFAULT #pylint: disable=line-too-long
+
+  def configure(self, #pylint: disable=arguments-differ
+                dtype: Optional[np.dtype] = None,
+                precision: Optional[Text] = None):
+    self._dtype = np.dtype(dtype) if dtype is not None else None
+    self.jax_precision = precision if precision is not None else libjax.lax.Precision.DEFAULT #pylint: disable=line-too-long
```

Comment:
you might need to force it to be HIGHEST

Reply:
I'd like to leave it at whatever the default value is in JAX